docs/content/latest/troubleshoot/cluster/replace_master.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Replace a failed YB-Master
+headerTitle: Replace a failed YB-Master
+linkTitle: Replace a failed YB-Master
+description: Steps to replace a failed YB-Master in a YugabyteDB cluster.
+aliases:
+  - /troubleshoot/cluster/replace-master/
+  - /latest/troubleshoot/cluster/replace-master/
+menu:
+  latest:
+    identifier: replace-failed-master
+    parent: troubleshoot-cluster
+    weight: 831
+isTocNested: true
+showAsideToc: true
+---
+
+To replace a failed YB-Master server in a YugabyteDB cluster, follow these steps:
+
+For the steps below, the examples use the following scenario:
+
+- The cluster includes three `yb-master` servers: `M1`, `M2`, `M3`.
+- YB-Master server `M1` failed and needs to be replaced.
+- A new YB-Master server (`M4`) will replace `M1`.
+- The default master RPC port is `7100`
+
+
+1. Start the new (replacement) YB-Master server in standby mode. 
```

Comment:
I recall we had a conversation about this, but did we agree to go with `standby` as the naming for this state, publicly?

Reply:
Do you mean `replacement` -> `standby` ?